javascripts/map.js

```diff
@@ -1519,8 +1519,11 @@ function printDirections(){
 
 //determine which stylesheet to use, mobile or desktop
 function isMobile() {
-  try{ document.createEvent("TouchEvent"); return true; }
-  catch(e){ return false; }
+	if (navigator.userAgent.match(/Mobi/))
```

Comment:
Consider maybe `window.matchMedia` [1](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia),[2](http://caniuse.com/#search=matchmedia).  This regex isn't [nearly complicated enough](http://detectmobilebrowsers.com/) to detect all mobile browses.  That is, assuming that the reason you're detecting mobile is because they have small screens.